hack/bundle-cache-data.sh

```diff
@@ -0,0 +1,67 @@
+# extract select data from bundles:
+#  -env vars from all service operators
+#  -dataplane-operator bundle is cached (in order to merge at build time)
+set -ex
+
+function extract_bundle {
+    local IN_DIR=$1
+    local OUT_DIR=$2
+    for X in $(file ${IN_DIR}/* | grep gzip | cut -f 1 -d ':'); do
+        tar xvf $X -C ${OUT_DIR}/;
+    done
+}
+
+function extract_csv {
+    local IN_DIR=$1
+    local OUT_DIR=$2
+    for X in $(file ${IN_DIR}/* | grep gzip | cut -f 1 -d ':'); do
+        tar xvf $X -C ${OUT_DIR} --wildcards --no-anchor '**/*clusterserviceversion.yaml'
+    done
+}
+
+OUT_BUNDLE=bundle_extra_data
```

Comment:
do we want to put this dir into .gitignore

Reply:
we can for now. In a future PR I was considering making this something we generated and commit on each PR. That would require that we get it working with Renovate and other CI actions as well.

Reply:
added the .gitignore in the latest commit

Reply:
Thanks. I agree.